java-2016/homework-9/IOhomework/src/WriteFile.java

```diff
@@ -0,0 +1,49 @@
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+import java.util.logging.Logger;
+
+/**
+ * Created by Turian Ovidiu.
+ * This class is responsible to write a file with an list of Athletes sorted by the ski time result.
+ */
+public class WriteFile {
+
+
+    private static Logger LOGGER = Logger.getLogger("WriteFile");
+
+
+    /**
+     * This method will write an given file with an given list of Athletes.
+     * @param athletes the list we want to write in a file.
+     * @param file we want to write.
+     */
+    public void writeFile(List<Athlete> athletes , File file) {
+
+        LOGGER.info("Inside write file method.");
+
+        if (athletes == null){
+            throw new IllegalArgumentException("No athletes to write.");
+        }
+
+        try (BufferedWriter writer = new BufferedWriter(new java.io.FileWriter(file.getAbsolutePath()))) {
+
+            String columns = "AthleteNumber,AthleteName,CountryCode,SkiTimeResult,FirstShootingRange,SecondShooting,ThirdShootingRange";
+            writer.write(columns);
+            writer.write("\n");
+
+            for (Athlete athlete : athletes) {
```

Comment:
a nice toString method on the Athlete would have solved this concatenation issue. Also we talked about concatenating strings with + ...didn't we? :)